Sources/KlaviyoSwift/KlaviyoModels.swift

```diff
@@ -38,9 +38,9 @@ public struct Event: Equatable {
         public let email: String?
         public let phoneNumber: String?
         public let externalId: String?
-        init(email: String? = nil,
-             phoneNumber: String? = nil,
-             externalId: String? = nil) {
+        public init(email: String? = nil,
+                    phoneNumber: String? = nil,
```

Comment:
total nit, but the extra indentation seems like a bit much?

Reply:
I think that was swiftformat!